src/Application/Middleware/PersonRecaptchaMiddleware.php

```diff
@@ -32,4 +32,13 @@ protected function getCode(ServerRequestInterface $request): ?string
 
         return $captchaCode;
     }
+
+    protected function isUsingFriendlyCaptcha(ServerRequestInterface $request): bool
```

Comment:
Minor and maybe not worth it, but is it less confusing to rename these classes from ...Recaptcha... to ...Captcha... if they are becoming multi-service?

Reply:
Yes, but since the ticket is just for prototyping I thought I'd keep the old name for now, then rename if/when we decide to keep the friendly captcha.